junit5-support/src/main/kotlin/run/qontract/test/QontractJUnitSupport.kt

```diff
@@ -18,30 +18,39 @@ import kotlin.system.exitProcess
 val pass = Unit
 
 open class QontractJUnitSupport {
+    companion object {
```

Comment:
I prefer to avoid companion objects unless where necessary. Do you see any problem with moving these constants outside QontractJUnitSupport? They can sit by themselves without having to be in a class.

Reply:
These are constants that are related only to the JunitSupport Class. We can move them after we see a pattern.